mne/io/brainvision/brainvision.py

```diff
@@ -55,7 +55,7 @@ class RawBrainVision(_BaseRaw):
         triggers will be ignored. Default is 0 for backwards compatibility, but
         typically another value or None will be necessary.
     event_id : dict | None
-        The id of the event to consider. If None (default),
+        The id of special events to consider. If None (default),
```

Comment:
It should mention that it is an addition to regex. for instance, `event_id` for Epochs only selects the epochs that have an entry in `event_id`, otherwise it's not considered. I would say something like if dict, in addition to regex (probably not say that) we consider keys...